### PR TITLE
Fix linting errors in all meta.ymls

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

**Review by edmundmiller — APPROVED** [review]

File: modules/nf-core/basicpy/meta.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ tools:
       documentation: "https://basicpy.readthedocs.io/en/latest/index.html"
       tool_dev_url: "https://github.com/peng-lab/BaSiCPy"
       doi: 10.1038/ncomms14836
-      licence: "MIT License"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/basicpy/meta.yml:15]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/circexplorer2/annotate/meta.yml [hunk 2]
```diff
@@ -3,13 +3,14 @@ description: Annotate circRNAs detected in the output from CIRCexplorer2 parse
...
   - annotate
+  - circexplorer2
 tools:
   - "circexplorer2":
       description: "Circular RNA analysis toolkits"
       homepage: "https://github.com/YangLab/CIRCexplorer2/"
       documentation: "https://circexplorer2.readthedocs.io/en/latest/"
       doi: "10.1101/gr.202895.115"
-      licence: "['MIT License']"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/circexplorer2/annotate/meta.yml:13]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/circexplorer2/parse/meta.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ tools:
       homepage: "https://github.com/YangLab/CIRCexplorer2/"
       documentation: "https://circexplorer2.readthedocs.io/en/latest/"
       doi: "10.1101/gr.202895.115"
-      licence: "['MIT License']"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/circexplorer2/parse/meta.yml:13]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/coreograph/meta.yml [hunk 4]
```diff
@@ -12,7 +12,7 @@ tools:
       documentation: "https://mcmicro.org/troubleshooting/tuning/coreograph.html"
       tool_dev_url: "https://github.com/HMS-IDAC/UNetCoreograph"
       doi: 10.1038/s41592-021-01308-y
-      licence: "MIT License"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/coreograph/meta.yml:15]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/kallistobustools/count/meta.yml [hunk 5]
```diff
@@ -12,7 +12,7 @@ tools:
       homepage: https://www.kallistobus.tools/
       documentation: https://kb-python.readthedocs.io/en/latest/index.html
       tool_dev_url: https://github.com/pachterlab/kb_python
-      licence: MIT License
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/kallistobustools/count/meta.yml:15]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/kallistobustools/ref/meta.yml [hunk 6]
```diff
@@ -14,7 +14,7 @@ tools:
       documentation: https://kb-python.readthedocs.io/en/latest/index.html
       tool_dev_url: https://github.com/pachterlab/kb_python
       doi: "10.22002/D1.1876"
-      licence: MIT License
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/kallistobustools/ref/meta.yml:17]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/kofamscan/meta.yml [hunk 7]
```diff
@@ -11,7 +11,7 @@ tools:
       documentation: "https://github.com/takaram/kofam_scan"
       tool_dev_url: "https://github.com/takaram/kofam_scan"
       doi: "10.1093/bioinformatics/btz859"
-      licence: "['MIT License']"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/kofamscan/meta.yml:14]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/nanocomp/meta.yml [hunk 8]
```diff
@@ -11,7 +11,7 @@ tools:
       description: "Compare multiple runs of long read sequencing data and alignments"
       homepage: "https://github.com/wdecoster/nanocomp"
       documentation: "https://github.com/wdecoster/nanocomp"
-      licence: "MIT License"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/nanocomp/meta.yml:14]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/ngsbits/samplegender/meta.yml [hunk 9]
```diff
@@ -12,7 +12,7 @@ tools:
       homepage: "https://github.com/imgag/ngs-bits"
       documentation: "https://github.com/imgag/ngs-bits"
       tool_dev_url: "https://github.com/imgag/ngs-bits"
-      licence: "['MIT license']"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/ngsbits/samplegender/meta.yml:15]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/scimap/mcmicro/meta.yml [hunk 10]
```diff
@@ -10,7 +10,7 @@ tools:
       homepage: "https://scimap.xyz/"
       documentation: "https://scimap.xyz/All%20Functions/A.%20Pre%20Processing/sm.pp.mcmicro_to_scimap/"
       tool_dev_url: "https://github.com/labsyspharm/scimap"
-      licence: "MIT License"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/scimap/mcmicro/meta.yml:13]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/sickle/meta.yml [hunk 11]
```diff
@@ -9,7 +9,7 @@ tools:
       description: "a tool that determines clipping of reads on 3' end and 5'end based on base quality "
       homepage: "https://github.com/najoshi/sickle"
       tool_dev_url: "https://github.com/najoshi/sickle"
-      licence: "['MIT License']"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/sickle/meta.yml:12]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/somalier/extract/meta.yml [hunk 12]
```diff
@@ -19,7 +19,7 @@ tools:
       documentation: "https://github.com/brentp/somalier/blob/master/README.md"
       tool_dev_url: "https://github.com/brentp/somalier"
       doi: "10.1186/s13073-020-00761-2"
-      licence: "MIT License"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/somalier/extract/meta.yml:22]:
```suggestion
      licence: ["MIT"]
```

File: modules/nf-core/somalier/relate/meta.yml [hunk 13]
```diff
@@ -19,15 +19,15 @@ tools:
       documentation: "https://github.com/brentp/somalier/blob/master/README.md"
       tool_dev_url: "https://github.com/brentp/somalier"
       doi: "10.1186/s13073-020-00761-2"
-      licence: "MIT License"
+      licence: ["MIT License"]
```
Comment (maxulysse) [inline modules/nf-core/somalier/relate/meta.yml:22]:
```suggestion
      licence: ["MIT"]
```

**Review by maxulysse — COMMENTED** [review]
"MIT License" -> "MIT"

**Comment by mashehu** [comment]
This PR is too big for github actions (stratagy matrix > 256) 😞  will try to split it up

**Pull request closed without merging**